### ISSUE TITLE
Add start and stop dates to the readings chart in the monitoring panel

### ISSUE THREAD

**Issue by dharezlak** [ismop/eitwid#156]
* current zooming functionality of the chart should change the values of the dates (no more `Reset zoom` button),
* default date range should contain the last month period,
* modified time period should persist during subsequent data series choices,
* (try to combine this with #107).

**Comment by mkasztelnik** [comment]
I don't agree with first point. IMHO zoom in should be still available and behave as it is right now (so `Reset zoom` will reset into `from`-`to` values).

**Comment by dharezlak** [comment]
I specifically asked BBB about this and she confirmed that both mechanisms should be interchangeable.

**Comment by mkasztelnik** [comment]
If you are 100% sure they want this we can start with this approach, but I think that is not user friendly at all because it is too easy to zoom in and to hard to zoom out IMHO.
